docs/tasks/issuers/setup-acme/dns01/azuredns.rst

```diff
@@ -64,3 +64,5 @@ You can configure the issuer like so:
              # ResourceGroup name where dns zone is provisioned
              resourceGroupName: AZURE_RESOURCE_GROUP
              hostedZoneName: AZURE_DNS_ZONE_NAME
+             # Azure Cloud Environment, default to AZUREPUBLICCLOUD
+             environment: AZURE_ENVIRONMENT
```

Comment:
Can you update AZURE_ENVIRONMENT to a sensible default value? (e.g. `AzurePublicCloud`)

Reply:
it is better to keep the all capital name
as it is the same name used by azure and also used by other tools like external dns as well

Reply:
ok